Common subdirectories: kqemu-1.4.0pre1-orig/common and kqemu-1.4.0pre1/common
diff -u kqemu-1.4.0pre1-orig/kqemu-linux.c kqemu-1.4.0pre1/kqemu-linux.c
--- kqemu-1.4.0pre1-orig/kqemu-linux.c	2008-05-30 16:33:34.000000000 -0400
+++ kqemu-1.4.0pre1/kqemu-linux.c	2011-01-05 16:51:45.407394748 -0500
@@ -53,6 +53,10 @@
 #define PAGE_KERNEL_EXEC PAGE_KERNEL
 #endif
 
+/* init_MUTEX macro was removed in 2.6.37 */
+#if LINUX_VERSION_CODE > KERNEL_VERSION(2, 6, 36) && !defined(init_MUTEX)
+#define init_MUTEX(sem)		sema_init(sem, 1)
+#endif
 //#define DEBUG
 
 #ifdef DEBUG
Common subdirectories: kqemu-1.4.0pre1-orig/tests and kqemu-1.4.0pre1/tests
